app/main/db/queries.py

```diff
@@ -352,9 +388,17 @@ def build_browse_consignment_query(
             query = query.filter(date_filter)
 
     if sorting_orders:
-        query = _build_sorting_orders(query, sub_query, sorting_orders)
+        if "date_last_modified" in sorting_orders:
```

Comment:
not sure about this. The ticket didn't initially consider the sorting. I feel like if the sort dropdown says Date last modified and so the param we send here is `date_last_modified`, then  we shouldn't be getting end_date. Ideally we would change it to be consistent with what is shown in the table, Date of record, and then this all makes sense. Worth speaking to Terry about the dropdown.